src/data/ACTIONS/layers/patch7.05.ts

```diff
@@ -1,10 +1,244 @@
+import {iconUrl} from 'data/icon'
 import {Layer} from 'data/layer'
+import {Attribute} from 'event'
 import {ActionRoot} from '../root'
+import {SHARED} from '../root/SHARED'
 import {BonusModifier} from '../type'
 
 export const patch705: Layer<ActionRoot> = {
 	patch: '7.05',
 	data: {
+		// My Viper, WHY?!
+		STEEL_FANGS: {statusesApplied: ['HONED_REAVERS']},
+		DREAD_FANGS: {id: SHARED.UNKNOWN.id},
```

Comment:
Since this action was "removed" (by being renamed), the entire action should be replaced by `SHARED.UNKNOWN`
```suggestion
		DREAD_FANGS: SHARED.UNKNOWN,
```